go/pkg/cas/client.go

```diff
@@ -68,31 +68,36 @@ type ClientConfig struct {
 	// FileIOSize is the size of file reads.
 	FileIOSize int64
 
-	// FindMissingBlobsBatchSize is the maximum number of digests to check in a
-	// single FindMissingBlobs RPC.
-	FindMissingBlobsBatchSize int
-
-	// BatchUpdateBlobsConcurrency is the maximum number of BatchUpdateBlobs RPCs
-	// at the same time.
-	BatchUpdateBlobsConcurrency int
-
-	// MaxBatchTotalSizeBytes is the maximum number of blobs to upload/download
-	// in a single BatchUpdateBlobs/BatchReadBlobs RPC.
-	//
-	// If server capabilities are consulted (see IgnoreCapabilities), then
-	// this value is capped to the server-imposed limit.
-	// Thus Client.ClientConfig.MaxBatchTotalSizeBytes might be less than what
-	// was specified in NewClientWithConfig.
-	MaxBatchTotalSizeBytes int
+	// FindMissingBlobs is configuration for FindMissingBlobs RPCs.
+	// FindMissingBlobs.MaxSizeBytes is ignored.
+	FindMissingBlobs RPCConfig
+
+	// BatchUpdateBlobs is configuration for BatchUpdateBlobs RPCs.
+	BatchUpdateBlobs RPCConfig
 
 	// RetryPolicy specifies how to retry requests on transient errors.
 	RetryPolicy retry.BackoffPolicy
 
 	// IgnoreCapabilities specifies whether to ignore server-provided capabilities.
 	// Capabilities are consulted by default.
 	IgnoreCapabilities bool
+}
+
+// RPCConfig is configuration for a particular CAS RPC.
+// Some of the fields might not apply to certain RPCs.
+type RPCConfig struct {
+	// Concurrency is the maximum number of RPCs at a time.
+	Concurrency int
+
+	// MaxSizeBytes is the maximum size of the request/response, in bytes.
+	MaxSizeBytes int
 
-	// TODO(nodir): add per-RPC timeouts.
+	// MaxItems is the maximum number of blobs/digests per RPC.
+	// Applies to batch RPCs.
```

Comment:
`Applies only to batch RPCs` or `Ignored for Stream RPCs`.

Reply:
This is not about stream vs non-stream. 
This is about batch vs non-batch. An unary RPC can be non-batch too.

How can I say it more clearly, while saying concise?

Reply:
took a stab. PTAL.